### PR TITLE
Add Python in Education

### PR DESCRIPTION
**Pull request by quobit**
**https://github.com/quobit/awesome-python-in-education**

**This is a compilation of resources about Python in Education. Useful for teachers and students alike. Other programming languages like Swift in this awesome list have a link to similar content.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- The description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Please read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) more closely. Multiple points not followed from it. Try going through the list point for point and check that you follow it. :)

**Comment by quobit** [comment]
Ok. Checking... Thanks

**Comment by quobit** [comment]
Mmm... I think I need your help. I've added a description to the link but don't see any more items that apply. Could you please point them out?

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

File: readme.md [hunk 1]
```diff
@@ -125,6 +125,7 @@
 - [Python](https://github.com/vinta/awesome-python)
 	- [Asyncio](https://github.com/timofurrer/awesome-asyncio) - Asynchronous I/O in Python 3.
 	- [Scientific Audio](https://github.com/faroit/awesome-python-scientific-audio) - Scientific research in audio/music.
+	- [Education](https://github.com/quobit/awesome-python-in-education) - Resources about Python in Education.
```
Comment (sindresorhus) [inline readme.md]: The description is not very useful. It just repeats the title.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/quobit/awesome-python-in-education
✖ Linting

  README.md:406:3
  ✖    1:1    Exceeded max depth of 2 levels                                                                                                 remark-lint:awesome/toc
  ✖    1:1    Missing License section                                                                                                        remark-lint:awesome/license
  ✖    1:32   Invalid badge source                                                                                                           remark-lint:awesome/badge
  ✖   23:3    ToC item "Academic Papers" does not match corresponding heading "Reference"                                                    remark-lint:awesome/toc
  ✖   45:3    ToC item "BBC micro:bit" does not match corresponding heading "BBC microbit"                                                   remark-lint:awesome/toc
  ✖   55:56   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖   61:45   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖   62:37   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖   63:70   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖   72:48   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖   87:3    Link to https://forum.freecodecamp.com/t/lets-all-build-a-comprehensive-interactive-python-curriculum-together/103979 is dead  remark-lint:no-dead-urls
  ✖   87:186  List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖   95:94   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  106:375  List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  107:94   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  109:62   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  110:64   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  111:75   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  115:56   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  120:87   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  124:103  List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  130:87   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  137:96   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  139:56   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  147:3    Link to http://exercism.io/languages/python/ is dead                                                                           remark-lint:no-dead-urls
  ✖  149:33   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  157:56   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  161:63   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  163:185  List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  174:3    Link to https://www.python.org/doc/essays/cp4e/ is dead                                                                        remark-lint:no-dead-urls
  ✖  174:50   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  175:112  List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  176:146  List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  177:78   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  178:77   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  179:135  List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  180:230  List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  181:95   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  182:156  List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  188:58   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  192:54   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  195:35   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  196:168  List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  199:54   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  200:72   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  201:3    Link to http://edupython.co.uk/ is dead                                                                                        remark-lint:no-dead-urls
  ✖  213:64   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  214:54   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  215:130  List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  216:61   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  220:98   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  221:83   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  223:77   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  224:69   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  226:78   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  227:3    Link to http://pythonforbiologists.com/index.php/29-common-beginner-python-errors-on-one-page/ is dead                         remark-lint:no-dead-urls
  ✖  227:137  List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  233:63   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  234:3    Link to https://unsupervisedmethods.com/cheat-sheet-of-machine-learning-and-python-and-math-cheat-sheets-a4afe4e791b6 is dead  remark-lint:no-dead-urls
  ✖  235:73   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  238:55   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  242:3    Link to https://www.python.org/community/sigs/current/edu-sig/ is dead                                                         remark-lint:no-dead-urls
  ✖  242:75   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  243:69   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  244:73   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  245:58   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  246:70   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  250:3    Text "StackOverflow" should be written as "Stack Overflow"                                                                     remark-lint:awesome/spell-check
  ✖  255:51   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  256:66   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  257:58   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  259:74   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  263:28   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  264:45   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  265:31   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  266:35   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  271:52   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  273:35   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  277:33   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  278:35   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  279:70   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  280:28   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  281:66   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  282:37   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  283:48   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  284:65   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  285:76   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  286:92   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  287:36   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  288:35   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  289:47   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  292:37   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  293:60   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  297:36   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  298:29   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  299:60   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  304:88   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  307:44   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  308:46   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  310:43   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  311:45   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  312:49   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  313:58   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  314:47   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  315:54   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  316:66   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  317:44   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  322:55   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  323:78   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  324:3    Link to http://nb.bianp.net/sort/views/ is dead                                                                                remark-lint:no-dead-urls
  ✖  324:54   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  328:95   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  329:83   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  330:3    Link to http://jupyterlab-tutorial.readthedocs.io/ is dead                                                                     remark-lint:no-dead-urls
  ✖  331:33   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  333:57   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  342:70   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  344:89   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  346:89   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  353:31   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  354:29   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  355:47   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  356:49   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  357:64   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  361:33   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  362:55   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  363:29   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  366:57   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  367:31   List item description must start with valid casing                                                                             remark-lint:awesome/list-item
  ✖  375:50   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  376:44   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  377:28   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  378:38   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  379:69   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  380:53   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  381:53   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  386:54   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  387:3    Link to https://2016.pycon-au.org/programme/python_in_education_seminar is dead                                                remark-lint:no-dead-urls
  ✖  388:118  Text "youtube" should be written as "YouTube"                                                                                  remark-lint:awesome/spell-check
  ✖  393:188  List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  395:131  List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  396:166  List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  399:4    Text "youtube" should be written as "YouTube"                                                                                  remark-lint:awesome/spell-check
  ✖  399:78   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  400:4    Text "youtube" should be written as "YouTube"                                                                                  remark-lint:awesome/spell-check
  ✖  401:4    Text "youtube" should be written as "YouTube"                                                                                  remark-lint:awesome/spell-check
  ✖  403:114  List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  406:3    Don’t pad link with inner spaces                                                                                               remark-lint:no-inline-padding
  ✖  407:3    Link to https://thenewboston.com/videos.php?cat=98 is dead                                                                     remark-lint:no-dead-urls
  ✖  407:93   Link to https://thenewboston.com/videos.php?cat=362 is dead                                                                    remark-lint:no-dead-urls
  ✖  407:145  List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  414:125  List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  420:56   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  420:57   Text "github" should be written as "GitHub"                                                                                    remark-lint:awesome/spell-check
  ✖  423:139  List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  425:54   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  439:41   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item
  ✖  449:73   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  449:74   Found reference to undefined definition                                                                                        remark-lint:no-undefined-references
  ✖  463:77   List item link and description must be separated with a dash                                                                   remark-lint:awesome/list-item
  ✖  523:3    Link to https://microbit-playground.co.uk/ is dead                                                                             remark-lint:no-dead-urls
  ✖  535:61   List item description must end with proper punctuation                                                                         remark-lint:awesome/list-item

  162 errors
```

https://github.com/sindresorhus/awesome-lint

File: readme.md (hunk 1 above)
Comment (quobit) [reply readme.md]: Thank you. I'll write a proper one.

**Comment by quobit** [comment]
Thanks again. Given the linter output, I'll work it out.

**Comment by sindresorhus** [comment]
Bump

**Pull request closed without merging**